crates/typst-syntax/src/parser.rs

```diff
@@ -1562,8 +1564,8 @@ impl<'s> Parser<'s> {
         self.prev_end
     }
 
-    fn current(&self) -> SyntaxKind {
-        self.current
+    fn current_kind(&self) -> SyntaxKind {
```

Comment:
I think we could keep the name `fn current` for now. I would thus like to keep the diff as plain as possible for now since the parser isn't the focus.